fenix/app/src/main/java/org/mozilla/fenix/settings/CookieBannersSwitchPreference.kt

```diff
@@ -1,26 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-package org.mozilla.fenix.settings
-
-import android.content.Context
-import android.util.AttributeSet
-import org.mozilla.fenix.ext.settings
-
-/**
- * Cookie banners switch preference with a learn more link.
- */
-class CookieBannersSwitchPreference(context: Context, attrs: AttributeSet?) :
-    LearnMoreSwitchPreference(context, attrs) {
-
-    override fun getLearnMoreUrl(): String {
-        return SupportUtils.getGenericSumoURLForTopic(
-            SupportUtils.SumoTopic.COOKIE_BANNER,
```

Comment:
You should delete and SumoTopic.COOKIE_BANNER from SupportUtils because now it is not used.

Reply:
Yeah, let me update the PR!

Reply:
Updated!